bitnami/kafka/Chart.yaml

```diff
@@ -29,4 +29,4 @@ name: kafka
 sources:
   - https://github.com/bitnami/bitnami-docker-kafka
   - https://kafka.apache.org/
-version: 12.3.1
+version: 12.3.2
```

Comment:
Please bump minor version instead, since we're adding a new parameter.

Reply:
minor version added with the latest commit